assets/examples/web-components/dynamic-map.html

```diff
@@ -0,0 +1,164 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <!-- https://gist.github.com/dwnoble/9e3ac88e162248f849dd276ff5895ad0 -->
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://fonts.cdnfonts.com/css/museo-sans-rounded"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/bootstrap/4.6.2/css/bootstrap.min.css"
+    />
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/lodash.js/4.17.21/lodash.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.7.0/jquery.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/bootstrap/4.6.2/js/bootstrap.min.js"></script>
```

Comment:
are these libraries required for the examples? looks like you only need lodash and jquery

Reply:
@dwnoble do we need these libraries?

The bootstrap is used for navbar styling I believe.